messages/deploy.json

```diff
@@ -31,7 +31,7 @@
     "soapDeploy": "deploy metadata with SOAP API instead of REST API",
     "predestructivechanges": "file path for a manifest (destructiveChangesPre.xml) of components to delete before the deploy",
     "postdestructivechanges": "file path for a manifest (destructiveChangesPost.xml) of components to delete after the deploy",
-    "tracksource": "If the deploy succeeds, update the source tracking information, similar to push",
+    "tracksource": "If the deploy succeeds, update the source tracking information, similar to push.  This will NOT delete files that were deleted locally unless you use --predestructivechanges or --postdestructivechanges",
```

Comment:
Suggested change (I don't see the handy button to suggest a change, so adding this manually):

"if the deploy succeeds, update source tracking information; doesn't delete locally deleted files from org unless you also specify --predestructivechanges or --postdestructivechanges"

NOTE: In the interest of shortening the message, I remove the "similar to push" part -- I'll mention that in rel notes when this goes out.